### PR TITLE
Expand About page

### PR DESCRIPTION
**Pull request by vladh**
I think this gives potential members a much better and more detailed pitch.

cc @gusaus

### CODE REVIEW

**Comment by vladh** [comment]
Split off into #136.

**Comment by chadwhitacre** [comment]
The wall of text is intimidating. Can we adopt a FAQ format to make it more accessible?

**Comment by vladh** [comment]
Okay! On it.

**Comment by vladh** [comment]
@chadwhitacre Ready for re-review.

File: src/pages/about.astro [hunk 1]
```diff
@@ -3,29 +3,124 @@
...
+        <p
+          >Open Source Pledge is a group of companies with a shared commitment to paying the maintainers of the Open
+          Source software we all consume. Our goal is to establish a new social norm in the tech industry of companies
+          paying Open Source maintainers, so that burnout and related security issues such as those in XZ and Log4j can
+          become a thing of the past.</p
+        >
+      </section>
+
+      <section>
+        <h2>What does making the Pledge entail?</h2>
```
Comment (chadwhitacre) [inline src/pages/about.astro:43]: Let's keep this wording consistent.

```suggestion
        <h2>What does joining the Pledge entail?</h2>
```

File: src/pages/about.astro [hunk 2]
```diff
@@ -3,29 +3,124 @@
...
+        <p
+          >Our new norm is for companies to pay a minimum of $2000 per year per full-time equivalent developer on the
+          company's staff, to Open Source maintainers of each company's choosing.</p
+        >
+      </section>
+
+      <section>
+        <h2
+          >Can companies become eligible by hiring developers to work on Open Source software, or giving gifts such as
+          cloud computing credits?</h2
```
Comment (chadwhitacre) [inline src/pages/about.astro:54]: Kinda wordy. How about this?

```suggestion
          >What about other ways to support Open Source?</h2
```

Comment (vladh) [reply src/pages/about.astro]: @chadwhitacre See: https://github.com/opensourcepledge/opensourcepledge.com/pull/119/commits/57c90660e94f7fedec5e2ca45d46097d56a5c09d

**Comment by vladh** [comment]
@chadwhitacre Re our discussion, see: https://github.com/opensourcepledge/opensourcepledge.com/pull/119/commits/a7e34a481b74963bcf9696dca85b55e602bad901

**Comment by chadwhitacre** [comment]
Great iteration, we can build further from here. 👍

**Pull request merged**